ServerCore/Pages/Shared/_Layout.cshtml

```diff
@@ -16,11 +16,22 @@
         <link rel="stylesheet" href="~/css/site.min.css" asp-append-version="true" />
     </environment>
 
-    <!-- TODO: Make this actually scalable and correct -->
+    <base href="~/" />
+
     @{
         // Using ev here so that it isn't the same as Event that we bind in the <body>
         // Works fine if we use Event, but VS will yell at you with an error in the IDE
         var ev = (Model as ServerCore.ModelBases.EventSpecificPageModel)?.Event;
+    }
+
+    @if (ev == null || ev.AllowBlazor)

```

Comment:
For now let's not do this for the null event case so the ripcord is easier to pull.

Reply:
Done